### PR TITLE
Standardize spelling of text-to-speech throughout docs

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/google_cloud.markdown [hunk 1]
```diff
@@ -52,26 +52,26 @@ Basic instruction for all APIs:
     2. From the `Service account` list, select `New service account`.
     3. In the `Service account name` field, enter any name.
 
-    If you are requesting Text-to-Speech API key:
+    If you are requesting Text-to-speech API key:
```
Comment (c0ffeeca7) [inline source/_integrations/google_cloud.markdown]:
```suggestion
    If you are requesting a text-to-speech API key:
```

File: source/_integrations/google_cloud.markdown [hunk 2]
```diff
@@ -52,26 +52,26 @@ Basic instruction for all APIs:
...
-The Cloud Text-to-Speech API is priced monthly based on the amount of characters to synthesize into audio sent to the service.
+The Cloud text-to-speech API is priced monthly based on the amount of characters to synthesize into audio sent to the service.
 
 | Feature                       | Monthly free tier         | Paid usage                        |
 |-------------------------------|---------------------------|-----------------------------------|
 | Standard (non-WaveNet) voices | 0 to 4 million characters | $4.00 USD / 1 million characters  |
 | WaveNet voices                | 0 to 1 million characters | $16.00 USD / 1 million characters |
 
-### Text-to-Speech configuration
+### text-to-speech configuration
```
Comment (c0ffeeca7) [inline source/_integrations/google_cloud.markdown]:
```suggestion
### Text-to-speech configuration
```

**Review by c0ffeeca7 — COMMENTED** [review]
typos

**Review by frenck — APPROVED** [review]
Thanks, @c0ffeeca7 👍 

../Frenck

**Pull request merged**